website/src/releases/2020-10/release-v17.mdx

```diff
@@ -0,0 +1,233 @@
+---
+date: 2020-10-14
+title: Custom Applications v17
+description: |
+  The Application Kit packages have been released as a new major version v17.\
+  In this release note we go through the important migration steps.
+type: announcement
+topics:
+  - Breaking Changes
+  - Dependencies
+---
+
+The Application Kit packages have been released as a new **major version** `v17`. One of the most important changes in this release is about migrating to the new Apollo Client `v3`.
+
+Follow the steps below to migrate your Custom Application to the new versions.
+
+<!--more-->
+
+# Migrating to Apollo Client v3
+
+Migrating to the new Apollo Client `v3` requires some important migration steps.
+
+## Migrating Apollo imports
+
+Apollo ships now with a single package `@apollo/client` instead of multiple ones.
+
+For Custom Applications this means that the peer dependencies `apollo-client` and `react-apollo` are now replaced with the new peer dependency of `@apollo/client`.
```

Comment:
@emmenko Would nice to include code changes, just like you added in other sections of this release note.
```diff
-import { Query } from 'react-apollo';
+import { Query } from '@apollo/client/react/components';
```

Reply:
There is a link below to the official apollo docs, where it's properly documented how you change the imports.

I don't think we need to duplicate those examples here as well.